cranelift/codegen/src/isa/riscv64/inst/args.rs

```diff
@@ -127,15 +127,18 @@ impl AMode {
 
     /// Returns the registers that known to the register allocator.
```

Comment:
pre-existing typo but: s/that known/that are known/? or actually maybe refactor the sentence a bit e.g. "Provides operands within this AMode to the register allocator's operand collector."